### PR TITLE
Fix configuration issues

### PR DESCRIPTION
**Pull request by omerts**
Fix configuration mentioned in https://github.com/ros-planning/navigation2_tutorials/issues/56

### CODE REVIEW

**Comment by SteveMacenski** [comment]
@arubenecia can you review?

**Review by SteveMacenski — APPROVED** [review]

**Comment by elgarbe** [comment]
Hi, I've tested this changes on ros2 iron (WSL on win11) and I can't get [this tutorial](https://navigation.ros.org/setup_guides/sensors/setup_sensors.html) working. 
In the first terminal I run `ros2 launch sam_bot_description display.launch.py` on a second one `ros2 launch slam_toolbox online_async_launch.py` finally `ros2 launch nav2_bringup navigation_launch.py`
Then on Rviz I select /global_costmap/costmap and get 'No map received' warning. I can see the /map produced by slam.

**Comment by TannerGilbert** [comment]
@SteveMacenski The problem with no costmap displaying when following the ["First-Time Robot Setup Guide"](https://docs.nav2.org/setup_guides/index.html) still occurs (at least for me when tested with ROS 2 Humble). Using this branch the problem sees to be fixed so can this be merged?

File: sam_bot_description/launch/display.launch.py [hunk 1]
```diff
@@ -12,13 +12,14 @@ def generate_launch_description():
     robot_state_publisher_node = launch_ros.actions.Node(
         package='robot_state_publisher',
         executable='robot_state_publisher',
-        parameters=[{'robot_description': Command(['xacro ', LaunchConfiguration('model')])}]
+        parameters=[{'robot_description': Command(['xacro ', LaunchConfiguration('model')])},{'use_sim_time': LaunchConfiguration('use_sim_time')}]
```
Comment (SteveMacenski) [inline sam_bot_description/launch/display.launch.py:15]:
```suggestion
        parameters=[{'robot_description': Command(['xacro ', LaunchConfiguration('model')])}, {'use_sim_time': LaunchConfiguration('use_sim_time')}]
```

**Review by SteveMacenski — APPROVED** [review]

**Comment by SteveMacenski** [comment]
Sorry about the delay - I agree thanks for the ping @TannerGilbert 

Sometimes things get lost in the queue

**Pull request merged**